docs/services/azure-services-guide.md

```diff
@@ -0,0 +1,530 @@
+# Deploying Wordpress in Azure
+
+This user guide will walk you through Wordpress application deployment using Crossplane managed resources and
+the official Wordpress Helm chart.
+
+
+## Pre-requisites
+
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest)
+    * Make sure to [login](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli?view=azure-cli-latest) after installation.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+    * kubectl also be installed using the Azure CLI: `az aks install-cli`
+* [Helm](https://docs.helm.sh/using_helm/), minimum version `v2.10.0+`.
+* [jq](https://stedolan.github.io/jq/) - commandline JSON processor `v1.5+`
+
+
+## Preparation
+This guide assumes that you have setup the Azure CLI and are logged in to your desired account.
+
+*Note: environment variables are used throughout this guide. You may use the values below or create your own.*
+
+```bash
+export RESOURCE_GROUP_NAME=myResourceGroup
+export RESOURCE_GROUP_LOCATION=eastus
+export AKS_NAME=myAKSCluster
+export AKS_NODE_COUNT=1
+export AKS_RESOURCE_GROUP=MC_${RESOURCE_GROUP_NAME}_${AKS_NAME}_${RESOURCE_GROUP_LOCATION}
+export SUBSCRIPTION_ID=$(az account list | jq -j '.[0].id')
+``` 
+
+### AKS Cluster
+Azure maintains a succinct [walkthrough](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough) for setting up an AKS cluster using the Azure CLI. The basic steps are as follows:
+
+1. Create a Resource Group
+```bash
+az group create --name $RESOURCE_GROUP_NAME --location $RESOURCE_GROUP_LOCATION
+```
+
+2. Create AKS Cluster (this may take a few minutes)
+```bash
+az aks create \
+    --resource-group $RESOURCE_GROUP_NAME \
+    --name $AKS_NAME \
+    --node-count $AKS_NODE_COUNT \
+    --enable-addons monitoring \
+    --generate-ssh-keys
+```
+
+3. Enable SQL Service Endpoint
+
+Get name of AKS node Virtual Network:
+```bash
+export AKS_VNET=$(az network vnet list -g $AKS_RESOURCE_GROUP | jq -j '.[0].name')
+```
+
+Add Service Endpoint to AKS subnet:
+```bash
+az network vnet subnet update -g $AKS_RESOURCE_GROUP --vnet-name $AKS_VNET -n aks-subnet --service-endpoints Microsoft.Sql
+```
+
+4. Connect to AKS Cluster 
+```bash
+az aks get-credentials --resource-group $RESOURCE_GROUP_NAME --name $AKS_NAME
+```
+
+5. Make sure `kubectl` is able to communicate with AKS Cluster
+```bash
+kubectl get pods --all-namespaces
```

Comment:
nitpick: I have seen `kubectl cluster-info` command as a more general form for testing that kubectl is configured.